firefox-ios/Client/Frontend/Home/TopSites/DataManagement/TopSitesDataAdaptor.swift

```diff
@@ -49,13 +50,15 @@ class TopSitesDataAdaptorImplementation: TopSitesDataAdaptor, FeatureFlaggable {
          topSiteHistoryManager: TopSiteHistoryManager,
          googleTopSiteManager: GoogleTopSiteManager,
          contileProvider: ContileProviderInterface = ContileProvider(),
+         unifiedAdsProvider: UnifiedAdsProviderInterface = UnifiedAdsProvider(),
          notificationCenter: NotificationProtocol = NotificationCenter.default,
```

Comment:
Not related to this PR, but I see us using `FooInterface` and `FooProtocol` interchangeably, whereas in Swift they will be both protocols. Is there a semantic difference, or are we just not aligned on a name?

Reply:
Great point, I've used both naming convention and don't feel attached to one or the other, but I can see value to sticking to one in particular. If you want it's worth bringing to the broader team for discussion 👍